armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -242,22 +305,8 @@ Type=simple
 WantedBy=multi-user.target
 EOF
 
-## customize ssh login message
+## disable ssh login messages
```

Comment:
Why remove it?

Reply:
It's a hassle to adjust it on the readonly file system when switching btw mainnet and testnet and not really needed. We can keep it if you think it's worth it, though.

Reply:
Fine by me to remove it then.